openpdf/src/main/java/com/lowagie/text/pdf/FontSelector.java

```diff
@@ -67,18 +67,40 @@
 
     protected ArrayList<Font> fonts = new ArrayList<>();
 
+    public FontSelector() {
+        FontFactory.register("font-fallback/LiberationSans-Regular.ttf", "sans");
+        Font font = FontFactory.getFont("sans", BaseFont.IDENTITY_H);
+        fonts.add(font);
+    }
+
+    /**
+     * change the color of default font in <CODE>FontSelector</CODE>.
+     * @param color the <CODE>Color</CODE> of default font
+     */
+    public void setDefaultColor(Color color) {
```

Comment:
Are you sure, that fonts.size() - 1 is always the DefaultFont?

Reply:
yes, fonts.size() - 1 is always the DefaultFont.